### ISSUE TITLE
Support SAS in ingestion when possible (ease of use)

### ISSUE THREAD

**Issue by vplauzon** [Azure/kusto-mirror-table#10]

**Comment by vplauzon** [comment]
In order to do that, we would need to first fetch the access keys of the storage account (if user has enough privilege).  This would require us to use Azure Management API (https://learn.microsoft.com/en-us/rest/api/storagerp/storage-accounts/list-keys?tabs=HTTP) which in turn would require us to know the subscription & resource group of the storage account (resource group could potentially be inferred by listing the storage accounts of a subscription, cf https://learn.microsoft.com/en-us/rest/api/storagerp/storage-accounts/list?tabs=HTTP).

Basically, it isn't trivial to implement without requiring more parameters from the user.
